nixos/modules/testing/service-runner.nix

```diff
@@ -121,7 +122,7 @@ in
 {
   options = {
     systemd.services = mkOption {
-      type = with types; attrsOf (submodule opts);
+      type = attrsOf (submodule opts);
```

Comment:
This usage of lib is totally fine for me and I see a lot of people doing it when writing modules. IMO we should keep it.